docs/operate-a-stake-pool/cardano-relay-configuration.md

```diff
@@ -220,3 +221,9 @@ Dec  1 15:31:40 localhost cardano-testnode[162851]: #033[35m[localhos:cardano.no
 ```
 
 Syncing the blockchain from zero can take a while. Please be patient. If you want to stop syncing, you can do so using the command `sudo systemctl stop cardano-node`. Restarting the relay node will resume syncing the blockchain.
+
+## Reloading the Topology configuration
+
+The topology configuration file controls the relays your node will connect to (eg. mainnet-topology.json). If you have made updates to this configuration you can load these changes without having to perform a full node restart by typing:
```

Comment:
The naming convention for filename on this page is topology.json (format of which is explained earlier on page). Also, on mainnet - recommendation is still to use P2P only on one of the relays. Hence, suggestion below:

``` suggestion
In case you have made an update to `topology.json` file, since this node is assumed to be running in P2P mode - you can can load these changes without having to perform a full node restart using command below:
```

Reply:
Likewise @jadjei I believe sending the reload signal is an essential part of P2P configuration even if all pool nodes aren't running it.  It takes too long to build a list of hot peers from scratch after a node restart & newer pool operators might suffer from not knowing this.  So I'd be in favour of making your changes if amended as @rdlrt suggests.

Reply:
> as being able to reload topology without a node restart is useful to all node

HUP signal is only applicable to nodes running P2P

Reply:
Ah OK .. good to know!